not-to-release/sources/answers/20111108082040AAmrGJ5_ans.xml.conllu

```diff
@@ -5,7 +5,7 @@
 1	Any	any	DET	DT	_	2	det	2:det	_
 2	information	information	NOUN	NN	Number=Sing	0	root	0:root	_
 3	about	about	ADP	IN	_	6	case	6:case	_
-4	CRAZY	CRAZY	PROPN	NNP	Number=Sing	5	compound	5:compound	_
+4	CRAZY	CRAZY	ADJ	NNP	Degree=Pos	5	amod	5:amod	_
```

Comment:
Lemmas should have title case?

Reply:
No I backed off of changing the case in the lemma

Reply:
OK, for the record these are all title-cased when appropriate in GUM

Reply:
Yeah, I think eventually we should go through all the NNPs and make the casing consistent. But doing it now for just the non-PROPN NNPs would make things more confusing I think.